src/components/MediaList/MediaList.tsx

```diff
@@ -122,19 +103,11 @@ interface MediaListItemBodyProps {
   resourceType?: "video" | "image" | "audio" | "text" | "h5p" | "podcast";
 }
 
-const StyledMediaListItemBody = styled.div`
-  ${fonts.size.text.metaText.small};
-  ${mq.range({ from: breakpoints.tablet })} {
-    max-width: 70%;
-  }
-  ${mq.range({ from: breakpoints.desktop })} {
-    max-width: 75%;
-  }
-`;
+const StyledMediaListItemBody = styled("div", {
+  base: { tablet: { maxWidth: "70%" }, desktop: { maxWidth: "75%" } },
```

Comment:
Hvorfor trenger vi dette?

Reply:
Tenkte det var en grunn til at det var sånn før, så beholdt det, for min del kan det fjernes

Reply:
Tipper at vi klarer oss uten, hvis alt annet tar opp full plass